nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -224,6 +224,19 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `appimageTools.wrapAppImage` now creates the binary at `$out/bin/${pname}` rather than `$out/bin/${pname}-${version}`, which will break downstream workarounds.
 
+- `apptainer` and `singularity` now prioritize system-wide `PATH` over those constructed from dependent packages when searching for third-party utilities. The `PATH` to search for third-party utilities, known as `defaultPath` inside Apptainer/Singularity source code, is now constructed from the following sources, ordered by their precedence:
```

Comment:
My memory's a bit hazy, I now forget why we wanted the system PATH to take the priority?

Reply:
Because the `fusermount` (or `fusermount3`) executable needs its SUID bit set to be able to execute by unprivileged users.

Reply:
H'm I see how /run/wrapper/bin is correct, but I'm still dizzy about FHS

Reply:
But ok this isn't exactly the question for backporting

Reply:
> but I'm still dizzy as to why FHS isn't the last priority

That's indeed an unusual design choice for Nix packages.

As Apptainer and SingularityCE rely more and more on system-level utilities and even the ones with SUID bit set, it becomes much easier to utilize them from the environment instead of inventing more arguments like `newuidmapPath` and `newgidmapPath` that unprivileged users need to override to get Apptainer/SingularityCE to function correctly.